Mimir.Worker.Tests/Handler/AgentStateHandlerTests.cs

```diff
@@ -0,0 +1,46 @@
+using Bencodex;
+using Libplanet.Crypto;
+using Mimir.Worker.Handler;
+using Mimir.Worker.Models;
+
+namespace Mimir.Worker.Tests.Handler;
+
+public class AgentStateHandlerTests
+{
+    private static readonly Codec Codec = new();
+    private readonly AgentStateHandler _handler = new();
+
+    [Theory]
+    [InlineData(0)]
+    [InlineData(3)]
+    public void ConvertToStateData(int avatarCount)
+    {
+        var address = new PrivateKey().Address;
+        var agentState = new Nekoyume.Model.State.AgentState(address);
+        for (var i = 0; i < avatarCount; i++)
+        {
+            agentState.avatarAddresses.Add(i, new PrivateKey().Address);
+        }
+
+        var context = new StateDiffContext
+        {
+            Address = address,
+            RawState = agentState.SerializeList(),
```

Comment:
It may need to check when the `AgentState` is `Dictionary`

Reply:
I thought so too, and tried to serialize it as a `Dictionary` type, but the code in `Lib9c` has [changed](https://github.com/planetarium/lib9c/blob/main/Lib9c/Model/State/AgentState.cs#L73-L99), so it's not possible.
I think I can omit it since the state migration is already done, but what do you think?